### DISCUSSION TITLE
How to update components

### DISCUSSION THREAD

**Discussion by ftognetto** [huntabyte/shadcn-svelte#200] [Help]
### Change Type

Addition

### Proposed Changes

Hi @huntabyte i have a simple question, which is the correct way to get the last updates of components?

For example if a bug get fixed on select component but I have already add the select in my project, not being published as a npm package, should I delete the select and run add command again?

I searched the same question on shadcn repo but it seems there isn't an answer or a best practice about.

thanks!

**Comment by huntabyte** [comment]
If something changes within shadcn-svelte, you can either run the CLI again, or you can compare and make the adjustments. If the fix is in `bits-ui`, you can simply update the package. @ftognetto

**Comment by ftognetto** [comment]
Thank you. Do you mean to run the cli with --overwrite? `npx shadcn-svelte@latest add select --overwrite`
